static/app/components/events/interfaces/spans/spanTreeModel.tsx

```diff
@@ -530,6 +706,19 @@ class SpanTreeModel {
     this.showSpanGroup = !this.showSpanGroup;
   };
 
+  toggleSiblingSpanGroup = (
+    operation: string | undefined,
+    description: string | undefined
+  ) => {
+    const key = `${operation}.${description}`;
```

Comment:
@0Calories Minor nit: rather than passing `operation` and `description` around explicitly, you can just pass `span` object around to abstract out what it means for siblings to be grouped.

Then you can have a convenience function like the following to centralize the key generation:

```
function siblingGroupKey(span: SpanType): string {
    return `${span?.operation}.${span?.description}`;
}
```

Reply:
That's a good point, I'll change that 👍